photon-api/src/main/scala/com/linkedin/photon/ml/data/GameConverters.scala

```diff
@@ -81,7 +82,8 @@ object GameConverters {
 
     val featureShardContainer = featureShards.map { shardId =>
       val features = row.getAs[SparseVector](shardId)
-      (shardId, VectorUtils.mllibToBreeze(features))
+      val breezeVector = new BreezeSparseVector[Double](features.indices, features.values, features.size)
+      (shardId, breezeVector)
```

Comment:
We should add a TODO that this line should return to as it was before once `mllib.Vector` is replaced throughout all of Photon ML.